### DISCUSSION TITLE
An error occurred during credentials validation

### DISCUSSION THREAD

**Discussion by onepisYa** [langgenius/dify#3639] [General]
I read #2854 and #2624 、#2540

I have searched for existing issues, including closed ones.
I confirm that I am using English to submit this report

I am using https://cloud.dify.ai/apps

My computer is Mac.

1. I confirmed that Ollama is successfully running;
2. I disabled all firewall and now all incoming calls are allowed;
3. I set OLLAMA_HOST to 0.0.0.0
4. I am not using docker.
5. I restarted Ollama and rerun Ollama run llama2. I use `ollama serve` run serve.
6. I input llama2 in dify backend as model name and http://172.19.0.1:11434/ as base_url
I still got the same error.

```zsh
# ✅  I got a result.
curl http://localhost:11434/api/chat -d '{
  "model": "llama3",
  "messages": [
    { "role": "user", "content": "why is the sky blue?" }
  ]
}'
```

**Comment by MatheMatrix** [comment]
@onepisYa  hello, do resolve this issue? I'm facing same problem :(

**Reply by MatheMatrix** [reply]
I find it is because there are proxy configured in dify docker environment 😄

**Comment by mzh1996** [comment]
I found solution from https://github.com/langgenius/dify/issues/6609 and https://blog.csdn.net/xx_nm98/article/details/141143526
If you use MacOS and ollama, the LLM url in dify must be like http://host.docker.internal:11434, but not http://0.0.0.0:11434

**Reply by wydevise** [reply]
I'm running comfyui on a Mac, but when I fill in the port address in the Windows version of dify it doesn't work. Do you know a solution?
